site/app/controllers/student/SubmissionController.php

```diff
@@ -1423,7 +1423,7 @@ public function ajaxUploadSubmission($gradeable_id, $merge = null, $clobber = nu
             $vcs_base_url = $this->core->getConfig()->getVcsBaseUrl();
             $vcs_path = $gradeable->getVcsSubdirectory();
 
-            if ($gradeable->getVcsHostType() == 0 || $gradeable->getVcsHostType() == 1) {
+            if ($gradeable->getVcsHostType() == 0 || $gradeable->getVcsHostType() == 1 || $gradeable->getVcsHostType() == 4) {
```

Comment:
It would be preferable to use `===` here (for all three conditions, not just the new one)

Reply:
Changes have been made to the comparison operators.